python/pyspark/mllib/classification.py

```diff
@@ -207,7 +207,7 @@ def train(cls, data, iterations=100, initialWeights=None, regParam=0.01, regType
         """
         def train(rdd, i):
             return callMLlibFunc("trainLogisticRegressionModelWithLBFGS", rdd, int(iterations), i,
-                                 float(regParam), str(regType), bool(intercept), int(corrections),
+                                 float(regParam), regType, bool(intercept), int(corrections),
```

Comment:
I think directly use regType id enough, because py4j can translate "l1","l2",None in Python to "l1","l2",null in Java/Scala. 
And I found the peer functions LogisticRegressionWithSGD and SVMWithSGD also directly use regType and it can work well.